Migration-v11.md

```diff
@@ -0,0 +1,89 @@
+# Migration Guide for v11 (Brand Refresh)
+
+## Migration with Codemod
+
+**Step1:** Install this version of `@razorpay/blade`.
+
+**Step2:** Execute the codemod using the following command:
+
+> Need help? Check out [jscodeshift docs](https://github.com/facebook/jscodeshift) for CLI usage tips.
+
+```sh
+npx jscodeshift ./YOUR_DIR --extensions=tsx,ts,jsx,js -t ./node_modules/@razorpay/blade/codemods/brand-refresh/transformers/index.ts --ignore-pattern="**/node_modules/**"
+```
+
+> [!WARNING]
+>
+> While this codemod covers most cases, it's always good to double-check and test your code to catch any missed nuances.
+
+### 🚧 Watch Out for Limitations & Edge Cases
+
+> [!IMPORTANT]
+>
+> There might be some situations where the codemod falls short. If you encounter errors, handle those cases manually for a seamless transition. Happy coding!
+
+#### Typography
+
+1. The codemod doesn't handle the migration of conditionally rendered props. Take a moment to manually inspect and update such cases.
+
+   ```diff
+   - <Title size={isMobile ? "small" : "medium"}>hello world</Title>
+   + <Heading size={isMobile ? "large" : "xlarge"}>hello world</Heading>
+   ```
+
+## Manual Migration Guide
+
+### Typography Components
+
+- **The `Title` component has been removed in favor of the `Heading` component.**
+
+  ```diff
+  - <Title size="xlarge">hello world</Title>
+  + <Heading size="2xlarge">hello world</Heading>
```

Comment:
We can host it on the storybook, initially, I tried the same but the `diff` blocks aren't getting highlighted like markdown.